wiki/Rules/Code_of_conduct_for_modding_and_mapping/en.md

```diff
@@ -4,58 +4,32 @@ The **code of conduct for modding and mapping** is a set of rules and guidelines
-- When things look to be getting out of hand, or if the discussion is struggling to stay on topic, [call a moderator](/wiki/Reporting_bad_behaviour). Don't contribute to the problem by continuing to post.
-
-## Qualified beatmaps
+## Further exchange and discussion
 
-The [Qualified section](/wiki/Beatmap/Category#qualified) allows beatmaps to receive any final suggestions for change or improvement before reaching ranked status. When modding a qualified map, everything you consider "not minor" should **always** be mentioned — minor issues are also worth mentioning however, especially if they affect gameplay. If a qualified beatmap accumulates a significant number of minor issues, or if there is one or more major/unrankable issues, the beatmap should be disqualified so that fixes and improvements can be made before attempting requalification.
+Sometimes, discussions will be drawn-out, and a single mod and response isn't enough to bring all parties to agreement. Perhaps the mapper doesn't agree with a suggestion, and the modder isn't satisfied with their response. Ideally, all discussions should find closure within a reasonable timeframe.
 
-If you're unsure on the nature of a potential problem, ask for other opinions or review previous mod posts to make sure the "issue" presented isn't something done intentionally by the mapper — if so, it will likely have been discussed already. If there has been no prior discussion, feel free to open one as if you were modding the map normally, and post a brief outline of your concerns.
+Consider the following guidelines when an agreement cannot be reached:
 
-**Note that it is the community's responsibility alone to report issues with qualified beatmaps**. It is not guaranteed that any BN or NAT member will actively check and disqualify beatmaps without being prompted to do so.
+- **Ask if you're unsure about something.** Ask the person in question to explain what they meant. They should be more than willing to explain things to you in more detail.
+- **Ask someone with more experience, such as a member of [BN](/wiki/People/Beatmap_Nominators) or [NAT](/wiki/People/Nomination_Assessment_Team).** While they are busy people, they should be able to help you reach an agreement.
+- **Be open minded.** If you realise there's a more suitable alternative to something you originally suggested, don't be afraid of putting your pride aside.
+- **There are no wrong opinions** — it's up to the mapper and modders to come to an agreement in a civil manner. Try to understand alternative points of view rather than attack them, as you never know when it might be useful in the future.
+- **Take a break.** It is important to be at a right state of mind when discussing something. If you can't explain something through logic, you will not be able to explain it to them through violence either.
```

Comment:
first sentence more like "take a break [if the discussion is getting heated]" or better words idk
on its own, "take a break" feels kinda random because this obviously wouldn't help reach an agreement if there's no good reason to be taking a break lol

Reply:
Tried to be short and concise with that one. Second part of the review should tackle this.